components/builder-db/src/metrics.rs

```diff
@@ -34,6 +34,10 @@ impl metrics::Metric for Counter {
 
 pub enum Histogram {
     DbCallTime,
+    GetAllLatest,
```

Comment:
It might be good to have these names include a "CallTime" ... eg, "GetAllLatestCallTime"